packages/remix/package.json

```diff
@@ -26,9 +26,14 @@
       "types": "./build/types/index.types.d.ts",
       "browser": {
         "import": "./build/esm/index.client.js",
-        "require": "./build/cjs/index.client.js"
+        "require": "./build/cjs/index.client.js",
+        "default": "./build/cjs/index.client.js"
       },
-      "node": "./build/cjs/index.server.js"
+      "node": {
+        "import": "./build/esm/index.server.js",
```

Comment:
wondering if this `import` entry here is the reason for the failing tests 🤔

Reply:
Why do you think that? I am not sure myself - it should theoretically work like this and all our other Meta-Framework packages handle the exports like this.

Reply:
just because in SvelteKit we still use `node: "./build/cjs/index.server.js"` and it's still compatible with vite 6 

> ReferenceError: module is not defined in ES module scope

this suggested to me that we're using the wrong entry point somewhere, so I compared it to svetelkit. But not sure tbh, this could be a completely wrong idea too

Reply:
What seems to work is to set a new "worker" entry, and point it to our cloudflare build entries (similar to SvelteKit SDK).